pallets/parachain-staking/src/weights.rs

```diff
@@ -261,19 +261,14 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 			.saturating_add(T::DbWeight::get().reads(9 as Weight))
 			.saturating_add(T::DbWeight::get().writes(7 as Weight))
 	}
-	// If this takes up too much block space, run again on code
-	// - #743 benchmarks post reward payout optimization were 3x lower per collator,
-	// 15x lower per nominator
 	fn active_on_initialize(x: u32, y: u32) -> Weight {
```

Comment:
should rename to `passive_on_initialize -> on_round_transition_on_initialize()` and `active_on_initialize() -> payout_on_initialize` because we are no longer executing payouts at round transitions

Reply:
Renamed in 436beb9